prismic/custom-types/events.json

```diff
@@ -1,280 +1,265 @@
 {
-  "Main" : {
-    "uid" : {
-      "type" : "UID",
-      "fieldset" : "UID",
-      "config" : {
-        "placeholder" : "Leave blank to auto generate"
+  "Main": {
```

Comment:
Let's just test these changes on Prismic staging first and see if it'll destroy any existing data.